packages/grafana-ui/src/components/Table/LogViewCell.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { useStyles2 } from '../../themes';
+import { TableCellProps } from './types';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export function LogViewCell(props: TableCellProps): JSX.Element {
+  const { cell, tableStyles, cellProps } = props;
+  return (
+    <Tooltip placement="auto-start" content={<LogTooltip value={cell.value} />} theme="info-alt">
+      <div {...cellProps} className={tableStyles.cellContainer}>
```

Comment:
to define a custom cellContainer stylle you need to call tableStyles.buildCellContainerStyle and pass in a new option "noHover" or simlar, that builds a cell container style without the hover